content/src/content/jcr_root/apps/core/wcm/components/image/v2/image/clientlibs/site/.content.xml

```diff
@@ -2,4 +2,5 @@
 <jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
     jcr:primaryType="cq:ClientLibraryFolder"
     allowProxy="{Boolean}true"
-    categories="[core.wcm.components.image.v2]"/>
+    categories="[core.wcm.components.image.v2]"
+    jsProcessor="[default:none,min:gcc;languageIn=ECMASCRIPT_2015]"/>
```

Comment:
With this GCC will generate unnecessary polyfills as long as the languageOut will not be modified as well (https://github.com/google/closure-compiler/wiki/ECMAScript6). You should probably set languageOut as well to ES6.

Reply:
good point - added in last commit